src/components/accounts/EnterVerificationCode.js

```diff
@@ -173,16 +188,20 @@ const EnterVerificationCode = ({
                 >
                     <Translate id='button.verifyCodeEnable'/>
                 </FormButton>
+                <FormButton
+                    className='link red'
+                    onClick={onGoBack}
+                >
+                    <Translate id='button.cancel'/>
+                </FormButton>
             </form>
 
             <div className='resend'>
                 <div><Translate id='setRecoveryConfirm.didNotRecive'/></div>
                 <div>
-                    <span onClick={onResend} className='link'><Translate id='setRecoveryConfirm.resendCode'/></span>
-                    &nbsp;<Translate id='setRecoveryConfirm.or'/>&nbsp;<span onClick={onGoBack}
-                                                                             className='link'><Translate
-                    id='setRecoveryConfirm.sendToDifferent'/> <Translate
-                    id={`setRecoveryConfirm.${useEmail ? 'email' : 'phone'}`}/></span>
+                    <span onClick={!reSending ? onResend : () => {}} className='link'><Translate id={`setRecoveryConfirm.${!reSending ? 'resendCode' : 'resending'}`}/></span>
```

Comment:
What is the difference between `loading` and `reSending` props? Feels weird that `sending={loading}` on L171 but we use `reSending` here.  Should L171 also be referencing `reSending` for consistency?

Reply:
@MaximusHaximus yep good point. We only want the `loading` prop on that button to be true if we're either setting up a recovery message for a new account or creating an account + setting up a recovery method. The `reSending` flag can therefore not be used there since we don't want the button to show the 'Verifying...' string if the user is only re-sending the code. I made some changes here to clarify what the flag is doing and to make sure the button is only loading when it should be: https://github.com/near/near-wallet/pull/1774/commits/965f76cedc0ef83c720ed12205aa3611f473ba14